agent/config/types.go

```diff
@@ -98,6 +99,10 @@ type Config struct {
 	// AppArmorCapable specifies whether the Agent is capable of using AppArmor
 	// security options
 	AppArmorCapable bool
+
+	// TaskCleanupWaitDuration specifies the time to wait after a task is stopped
+	// until cleanup of task resources is started.
+	TaskCleanupWaitDuration time.Duration
```

Comment:
I thought about that and reconciled that a Task is essentially a runtime construct and when you cleanup a task, you clean up resources that are related to that. It's similar to docker rm cleaning up only containers.